rxbinding-support-recyclerview/src/androidTest/AndroidManifest.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<manifest package="com.jakewharton.rxbinding.support.v7"
```

Comment:
Append `.recyclerview`. The convention here is to replace `android.` with `com.jakewharton.rxbinding.` from the library's package name.